src/spring/azext_spring/_params.py

```diff
@@ -156,7 +156,14 @@ def load_arguments(self, _):
         c.argument('enable_application_configuration_service',
                    action='store_true',
                    options_list=['--enable-application-configuration-service', '--enable-acs'],
+                   arg_group="Application Configuration Service",
                    help='(Enterprise Tier Only) Enable Application Configuration Service.')
+        c.argument('application_configuration_service_generation',
+                   arg_group="Application Configuration Service",
+                   type=ConfigurationServiceGeneration,
+                   options_list=['--application-configuration-service-generation', '--acs-gen'],
+                   validator=validate_acs_create,
+                   help='(Enterprise Tier Only) Application Configuration Service Generation to enable. Allowed values are: ' + ', '.join(list(ConfigurationServiceGeneration)))
         c.argument('enable_application_live_view',
```

Comment:
If this parameter has limited values, please add `arg_type=get_enum_type(ConfigurationServiceGeneration)` for the parameter and it is not necessay to specify the avaiable values manually in `help`.
```suggestion
        c.argument('application_configuration_service_generation',
                   arg_group="Application Configuration Service",
                   arg_type=get_enum_type(ConfigurationServiceGeneration),
                   options_list=['--application-configuration-service-generation', '--acs-gen'],
                   validator=validate_acs_create,
                   help='(Enterprise Tier Only) Application Configuration Service Generation to enable.)
        c.argument('enable_application_live_view',
```

Reply:
Thanks! updated